modules/broker-auth/src/main/java/io/ballerina/messaging/broker/auth/authorization/enums/ResourceActions.java

```diff
@@ -16,23 +16,30 @@
  * under the License.
  *
  */
-package io.ballerina.messaging.broker.auth;
-
-import javax.security.auth.login.LoginException;
+package io.ballerina.messaging.broker.auth.authorization.enums;
 
 /**
- * This Exception class represents login failures.
+ * Enum used to represent resource actions.
  */
-public class BrokerAuthException extends LoginException {
+public enum ResourceActions {
```

Comment:
Shall we make the name Singular?
eg: ResourceAction

Reply:
Fixed in b30ae14